mits/mariadb_test.go

```diff
@@ -17,15 +17,114 @@
 package mits_test
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"strconv"
+
 	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+
+	"github.com/SUSE/minibroker-integration-tests/mits"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/generator"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/workflowhelpers"
 )
 
 var _ = Describe("MariaDB", func() {
-	testConfig := testsConfig.MariaDB
-
 	BeforeEach(func() {
-		if !testConfig.Enabled {
+		if !tests.MariaDB.Enabled {
 			Skip("Test is disabled")
 		}
 	})
+
+	It("should deploy and connect", func() {
+		orgName := testSetup.TestSpace.OrganizationName()
+		spaceName := testSetup.TestSpace.SpaceName()
+		appName := generator.PrefixedRandomName(tests.MariaDB.Class, "app")
+		serviceName := generator.PrefixedRandomName(tests.MariaDB.Class, "service")
+		securityGroupName := generator.PrefixedRandomName(tests.MariaDB.Class, "security-group")
+
+		By("pushing the test app without starting")
+		Expect(cf.Cf("push", appName, "--no-start", "-p", "assets/mariadbapp").Wait(timeouts.CFPush.Duration())).
```

Comment:
This is a bit long (probably because of the GitHub review interface); would it make sense to wrap before `Wait`? (I actually can't tell, because I'm not in a proper editor…)